src/Artsy/Analytics/useTracking.ts

```diff
@@ -0,0 +1,12 @@
+import * as AnalyticsSchema from "Artsy/Analytics/Schema"
+import { TrackingContext } from "Artsy/Analytics/TrackingContext"
+import { useContext } from "react"
+
+export function useTracking() {
+  const { tracking } = useContext(TrackingContext)
```

Comment:
What happens if there's no tracking context in the tree? What should happen? I'd vote for 'it should still work'.